### ISSUE TITLE
Wargame Title

### ISSUE THREAD

**Issue by ShendMikullovci** [serge-web/serge#1281]
Creating a wargame gives the latter a title containing a random number of letters. Do the letters indicate anything?

![Wargame Creation Screen 1](https://user-images.githubusercontent.com/85491794/147098969-868d6736-d55b-4fd2-9800-52abfcf7d8e0.png)
![Wargame Menu](https://user-images.githubusercontent.com/85491794/147098986-26b48328-95b8-40a7-8b8f-09ba0036c885.png)

If not, I suggest we show enumerated wargame titles such as:

-Wargame 3
-Wargame 2
-Wargame 1

With the newest one at the top of the list.

**Comment by IanMayo** [comment]
Wargames a given a random set of letters for a name - to ensure they are unique.  This name is used for the filename.

When the wargame is first opened, the name is editable - so Game Designers create a suitable name for it:
* "Atlantic Wargame"
* "Pacific Wargame"
* etc

**Comment by ShendMikullovci** [comment]
Understood. Can it be enumerated titles instead? They would still be unique in terms of file properties.

**Comment by IanMayo** [comment]
It's a challenge to check the number counters.  An alternative implementation (that avoids showing the random name) would be for `Create` to popup a dialog asking for a name for the wargame.  This would be shown in the "Edit wargame" page.

Would that be acceptable?

**Comment by ShendMikullovci** [comment]
Perfect.

**Comment by ShendMikullovci** [comment]
I also noticed that this same sequence of random letters appears for new force titles. Can we apply that same solution?
![Force Title](https://user-images.githubusercontent.com/85491794/147101533-037cea69-505e-4763-a56a-bcdcd3f1cff0.png)

**Comment by IanMayo** [comment]
> I also noticed that this same sequence of random letters appears for new force titles. Can we apply that same solution?

Yes, we could do.  But, it's a fairly low priority.  These pages are only accessed by the Game Designer, who should be familiar with Serge.
